### PR TITLE
PMM-5206: dashboards styling fixed

### PR DESCRIPTION
**Pull request by lunaticusgreen**
https://github.com/Percona-Lab/pmm-submodules/pull/592

For some mysterious reason we had a problem with panel-container styling after navigation from left grafana menu, to prevent it in the future, panel-container wrapper will be styled now with special class - custom-grafana-plugin

### CODE REVIEW

File: pmm-app/src/pmm-settings/panel.tsx [hunk 1]
```diff
@@ -28,12 +29,13 @@ const customCollapseStyle = {
 };
 
 const SettingsPanel = () => {
+  // document.getElementById('antd').classList.add('custom-grafana-plugin');
```
Comment (BupycHuk) [inline pmm-app/src/pmm-settings/panel.tsx]: do we need it?

File: pmm-app/src/react-plugins-deps/components/helpers/styling.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class Styling {
+  static setPluginPanelClass(panelWrapperId = '#antd', newClass = 'custom-grafana-plugin') {
```
Comment (BupycHuk) [inline pmm-app/src/react-plugins-deps/components/helpers/styling.ts]: maybe `panelWrapperSelector`?

Comment (BupycHuk) [inline pmm-app/src/react-plugins-deps/components/helpers/styling.ts]: looks like `addPluginPanelClass` fits better for this method. Set usually means replace old value with a new one.

**Comment by puneet0191** [comment]
Ready for Merge

**Pull request merged**